src/render/OpenGL.cpp

```diff
@@ -108,9 +119,21 @@ GLuint CHyprOpenGLImpl::compileShader(const GLuint& type, std::string src, bool
 
     GLint ok;
     glGetShaderiv(shader, GL_COMPILE_STATUS, &ok);
+
     if (dynamic) {
-        if (ok == GL_FALSE)
+        if (ok == GL_FALSE) {
+            GLint maxLength = 0;
+            glGetShaderiv(shader, GL_INFO_LOG_LENGTH, &maxLength);
+
+            // The maxLength includes the NULL character
+            std::vector<GLchar> errorLog(maxLength);
+            glGetShaderInfoLog(shader, maxLength, &maxLength, &errorLog[0]);
```

Comment:
oh god, `.data()` exists, man

Reply:
ah i was just grabbing the example khronos provided for openGL error processing, i see

Reply:
This should also be fixed, forgot to comment